webpack.config.babel.js

```diff
@@ -259,7 +259,13 @@ const configDist = {
         [new TerserPlugin({
           cache: true,
           parallel: true,
-          sourceMap: true
+          sourceMap: true,
+          terserOptions: {
+            output: {
+              preamble: `/* Onfido SDK ${packageJson.version} */`,
```

Comment:
What is this for?

Reply:
It adds a comment at the start of the minified file(s) with the `package.json` version number, 
so an example of the generated string would be `/* Onfido SDK 5.2.1 */`

Reply:
I'd like to add it into the minified CSS file too, but I've not found a way to do it.

Reply:
nice!